data/human/weapons.txt

```diff
@@ -1442,6 +1473,49 @@ outfit "Gatling Gun"
 		"hit effect" "bullet impact"
 	description `When Dr. Richard J. Gatling invented the rapid-fire gun bearing his name in the 19th century, he hoped that its destructiveness would lead to the end of war itself. Over a millennium later, this modern incarnation of the Gatling Gun sees action in the stars, a testament to its fundamentally robust design - and Dr. Gatling's naivete. While capable of a high fire rate, this gun needs to cool down for twice as long as it fires, and can only sustain a maximum of three seconds of continuous fire.`
 
+outfit "Gatling Turret"
+	category "Secondary Weapons"
+	cost 120000
+	thumbnail "outfit/gat turret"
+	"mass" 11
+	"outfit space" -23
+	"weapon capacity" -23
+	"turret mounts" -1
+	"gatling round capacity" 6000
+	weapon
+		sound "gatling"
+		"hardpoint sprite" "hardpoint/gat turret hardpoint"
+		"submunition" "Gat Submunition"
+			"offset" 1.5 -6
+			"facing" 0.3
+		"submunition" "Gat Submunition"
+			"offset" -1.5 -6
+			"facing" -0.3
+		ammo "Gatling Gun Ammo" 2
+		icon "icon/gat"
+		"inaccuracy" 2
+		"turret turn" 3.4
+		"velocity override" 16
+		"lifetime" 0
+		"reload" 3
+		"burst count" 180
+		"burst reload" 1
+		"firing energy" .06
+		"firing heat" 5
+		"cluster"
+		"hit effect" "bullet impact"
+	description `In an act that would make Dr. Richard J. Gatling spin in his grave with the ferocity of the barrels of his invention, the Gatling Turret was born when a resourceful pirate thought to replace the Energy Blasters on a Blaster Turret with a pair of Gatling Guns. Ever since, the dominion of the Gatling Turret has spread across the Southern Rim, terrorizing merchants and law enforcement alike with its unparalleled ability to tear apart the hulls of ships.`
+
+
+
+outfit "Gat Submunition"
+	weapon
+		sprite "projectile/tinyflare"
+		"hit effect" "bullet impact"
+		"velocity" 16
+		"lifetime" 1
+		"submunition" "gbullet"
```

Comment:
This adds an extra frame of delay as to when the visible projectiles are created by having a one-frame, zero velocity projectile be what spawns these submunitions that have the actual velocity, which can throw off the aim of the turret. Since the purpose of this is to have two tinyflare sprites appear, perhaps instead create a double tinyflare sprite that perfectly aligns with the barrels of the hardpoint sprite. The submunitions of the turret can then just be gbullet.